installation-usage.md

```diff
@@ -55,7 +55,25 @@ path inside of the kubelet root `/var/lib/kubelet/seccomp/operator`.
 
 ### 3. Apply profile to pod
 
-Create a pod using one of the created profiles:
+Create a pod using one of the created profiles. On Kubernetes >= 1.19, the
+profile can be specified as part of the pod's security context:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  containers:
+    - name: test-container
+      image: nginx
+      securityContext:
```

Comment:
nitpick: this would be adding it to a container, while the existing example is for the whole pod. Looks like the title of this section suggests adding to the pod. How would you feel about updating it to that?

Reply:
done

Reply:
Thanks!